@xen-orchestra/fs/src/s3.js

```diff
@@ -31,6 +32,11 @@ export default class S3Handler extends RemoteHandlerAbstract {
     if (protocol === 'http') {
       params.httpOptions.agent = new http.Agent()
       params.sslEnabled = false
+    } else if (protocol === 'https') {
+      params.httpOptions.agent = new https.Agent({
+        rejectUnauthorized: !allowUnauthorized,
```

Comment:
Should we create an agent if `allowUnauthorized` is not enabled?

If not provided, `https.request` will use `https.globalAgent` which:
- reuse opened connections if possible
- does not need to be manually destroyed when no longer necessary.

Reply:
I think the keepAlive parameter is important , and by default it's `false`

Reply:
Maybe but that's unrelated to this PR.

But I believe it's a good idea and I'm open to a PR for this in the future (for HTTP and HTTP both).

Reply:
Don't forget the PR for `keepAlive` :slightly_smiling_face: